bin/tasks.js

```diff
@@ -4,12 +4,12 @@ const path = require('path');
 const cwd = process.cwd();
 const target = path.resolve(cwd, 'public');
 const source = path.resolve(__dirname, '../src');
-const sass = require('witch')('node-sass');
+const npmsass = require('npm-sass');
```

Comment:
personally, I would prefer all of this to be in just plain ol boring JS code, include the modules, use them, etcetera. But I know you love your dirty bash